packages/tooling/src/lib/parameters-to-json-schema.js

```diff
@@ -21,7 +21,11 @@ function getBodyParam(pathOperation, oas) {
 
   const cleanupSchemaDefaults = obj => {
     Object.keys(obj).forEach(prop => {
-      if (typeof obj[prop] === 'object') {
+      if (obj[prop] === null) {
+        // If the item is null, just carry on. Why do this in addition to `typeof obj[prop] == object`? Because
+        // `typeof null` equates to `object` for "legacy reasons" apparently.
+        // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null
```

Comment:
lololol, Javascript

> ...because `typeof null` equates to `object` [for "legacy reasons"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null)

🤣🔫